src/auth.ts

```diff
@@ -55,7 +55,7 @@ export async function auth<
       } as GetWebFlowAuthorizationUrlOptions<Client>;
 
       if (options.clientType === "oauth-app") {
-        options.scopes = command.scopes ?? this.defaultScopes;
+        options.scopes = command.scopes ? this.defaultScopes : command.scopes;
```

Comment:
shouldn't this be the other way around? I think we could also do
```suggestion
        options.scopes = command.scopes || this.defaultScopes
```